crates/download/src/main.rs

```diff
@@ -132,34 +132,34 @@ async fn main() -> anyhow::Result<()> {
         output,
         version,
         additional_libraries_version,
-        accelerator,
+        device,
         cpu_arch,
         os,
     } = Args::parse();
 
     let octocrab = &octocrab()?;
 
     let core = find_gh_asset(octocrab, CORE_REPO_NAME, &version, |tag| {
-        let accelerator = match (os, accelerator) {
-            (Os::Linux, Accelerator::Cuda) => "gpu",
+        let device = match (os, device) {
+            (Os::Linux, Device::Cuda) => "gpu",
             (_, accelerator) => accelerator.into(),
```

Comment:
大きな問題はなさそうに思いますが、変数名を合わせると良さそうでした！

```suggestion
            (_, device) => device.into(),
```

Reply:
あー漏れてました。ありがとうございます。